src/lib/SlideBarChart.tsx

```diff
@@ -454,7 +465,7 @@ class SlideBarChart extends Component<SlideBarChartComponentProps, State> {
     } = this.props
 
     if (alwaysShowIndicator) {
-      this.moveCursorBinary(this.chartWidth / 2)
+      this.moveCursorBinary(this.chartWidth / 2, undefined, true);
```

Comment:
So in this cases, we could do:

```suggestion
      this.moveCursorBinary(this.getInitialIndicatorPosition());
```
Instead of sending an useless chartWidth.